docs/guide/dev/api_changes/README.md

```diff
@@ -46,7 +46,7 @@ The editor will now support **only one single skin per page** (all editors will
 
 Because of the above, the following skin-related properties were moved global or deleted:
 
- * {@linkapi CKEDITOR.skins CKEDITOR.skins} => {@linkapi CKEDITOR.skin CKEDITOR.skin}
+ * `CKEDITOR.skins` => {@linkapi CKEDITOR.skin CKEDITOR.skin}
```

Comment:
Why the redundancy in `{@linkapi CKEDITOR.skin CKEDITOR.skin}`? It could be simply `{@linkapi CKEDITOR.skin}`.

Reply:
There are more cases like that.